services/dashboard/app/routes/investments/$name/deal-team-highlights.tsx

```diff
@@ -21,71 +69,272 @@ interface LogEntry {
-    tags: [{ label: "Other", color: "gray" }],
-    author: "Monica Gomez Acebo",
-    date: "2025-01-01",
-  },
-  {
-    id: "log2",
-    title: "Integration for greater efficiency",
-    description:
-      "Positive feedback from the new CEO Oscar Garcia Maceiras, on Q1 performance. He was appointed CEO of Inditex on December 2024, succeding Carlos Crespo, as part of the broader leadership reshuffle aimed at ensuring continuity and accelerating Inditex´s global expansion and digital transformation.",
-    tags: [
-      { label: "Competition", color: "blue" },
-      { label: "Global Economy", color: "purple" },
-      { label: "Other", color: "gray" },
-    ],
-    author: "Monica Gomez Acebo",
-    date: "2024-10-20",
-  },
-  {
-    id: "log3",
-    title: "Internal Meeting",
-    description:
-      "Positive feedback from the new CEO Oscar Garcia Maceiras, on Q1 performance. He was appointed CEO of Inditex on December 2024, succeding Carlos Crespo, as part of the broader leadership reshuffle aimed at ensuring continuity and accelerating Inditex´s global expansion and digital transformation.",
-    tags: [{ label: "Other", color: "gray" }],
-    author: "Monica Gomez Acebo",
-    date: "2024-06-10",
-  },
-  {
-    id: "log4",
-    title: "Bank of America approach",
-    description:
-      "Positive feedback from the new CEO Oscar Garcia Maceiras, on Q1 performance. He was appointed CEO of Inditex on December 2024, succeding Carlos Crespo, as part of the broader leadership reshuffle aimed at ensuring continuity and accelerating Inditex´s global expansion and digital transformation.",
-    tags: [{ label: "Exit Strategy", color: "yellow" }],
-    author: "Monica Gomez Acebo",
-    date: "2024-03-23",
-  },
-];
+const colourClasses: Record<string, string> = {
+  blue: "bg-blue-100  text-blue-800",
+  green: "bg-green-100 text-green-800",
+  yellow: "bg-yellow-100 text-yellow-800",
+  gray: "bg-gray-200  text-gray-800",
+  red: "bg-red-100 text-red-800",
+};
 
 function TagBadge({ tag }: { tag: LogTag }) {
-  const bg = `${tag.color}-100`;
-  const text = `${tag.color}-800`;
   return (
     <span
-      className={`rounded-full px-2.5 py-0.5 text-xs font-medium bg-${bg} text-${text}`}
+      className={`rounded-full px-2.5 py-0.5 text-xs font-medium ${colourClasses[tag.color]}`}
     >
       {tag.label}
     </span>
   );
 }
 
+type LoaderData = {
+  labels: CommentLabel[];
+  comments: Comment[];
+};
+
+export const Route = createFileRoute("/investments/$name/deal-team-highlights")(
+  {
+    //Load data from server
+    loader: async (): Promise<LoaderData> => {
+      const [labels, comments] = await Promise.all([
+        getLabels(),
+        getComments(),
+      ]);
```

Comment:
These will make 2 requests to the server, why don't we replace it with just one network request?